src/tasks/task.ts

```diff
@@ -115,8 +114,8 @@ abstract class Task implements TaskInterface {
       .then(
         (runAndClientInfo): Beacon => this.generateBeacon(...runAndClientInfo)
       )
-      .then(this.encode)
-      .then(this.send)
+      .then((data): string => this.encode(data))
+      .then((data): void => this.send(data))
```

Comment:
Curious – why did you choose to make these changes?

Reply:
I can't actually remember :/ I think the compiler was complaining.